cypress/features/regression/experimental/dateInput.feature

```diff
@@ -103,4 +103,27 @@ Feature: Experimental Date Input component
   @positive
   Scenario: Check Date Input today date
     When I set dateInput to today
-    Then the date is set to today
\ No newline at end of file
+    Then the date is set to today
+
+  @positive
+  Scenario: Open dayPickerDay via click on input
+    When I click dateInput
+    Then dayPickerDay is visible
+
+  @positive
+  Scenario: Close dayPickerDay via click on input
+    Given I click dateInput
```

Comment:
Can you replace `given` and `when` to one step e.g `When I click twice on dateInput` ?

Reply:
updated